lib/quadquizaminos_web/templates/layout/root.html.leex

```diff
@@ -5,31 +5,58 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
     <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
     <%= csrf_meta_tag() %>
-    <%= live_title_tag assigns[:page_title] || "QuizQuadBlocks", suffix: " · SupplyChain Edition" %>
+    <%= live_title_tag assigns[:page_title] || "QuadBlocksQuiz", suffix: " · SupplyChain Edition" %>
     <link phx-track-static rel="stylesheet" href="<%= Routes.static_path(@conn, "/css/app.css") %>"/>
     <script defer phx-track-static type="text/javascript" src="<%= Routes.static_path(@conn, "/js/app.js") %>"></script>
   </head>
   <body>
     <header>
       <section class="container">
-        <nav role="navigation">
-          <ul>
-            <%= if @current_user do %>
-              <%= link "Logout", to: Routes.auth_path(QuadquizaminosWeb.Endpoint, :delete), method: "delete", class: "button" %>
-            <% end %>
-          </ul>
-        </nav>
-        <a href="https://sfractal.com/" class="phx-logo">
-          <img src="<%= Routes.static_path(@conn, "/images/sFractalConsulting.png") %>" alt="sFractal Logo" />
-        </a>
         <a href="https://sfractal.com/" class="phx-logo">
           <img src="<%= Routes.static_path(@conn, "/images/Supply Chain Sandbox logo_dark_draft.png") %>" alt="sFractal Logo" />
         </a>
-        <a href="https://podiihq.com/" class="phx-logo">
-          <img src="<%= Routes.static_path(@conn, "/images/podiilogo.png") %>" alt="Podii Logo" />
-        </a>
+        <H1>QuadBlocksQuiz</H1>
+        <H3>Supply Chain Edition</H3>
       </section>
     </header>
     <%= @inner_content %>
+<section class="container">
+  <nav role="navigation">
+    <ul>
+      <%= if @current_user do %>
+        <%= link "Logout", to: Routes.auth_path(QuadquizaminosWeb.Endpoint, :delete), method: "delete", class: "button" %>
+      <% end %>
+    </ul>
+  </nav>
+  <table>
+  <tr>
+    <td>
+      <a href="https://sfractal.com/" class="phx-logo">
```

Comment:
This looks good. A little adjustment will be needed, looking at the login page, the navbar appears at the bottom, and the layout looks good

![image](https://user-images.githubusercontent.com/43263401/112616570-f1bbea80-8e34-11eb-9a49-0998e02af26a.png)

But when we go to Tetris page and home page, the page layout has been affected

![image](https://user-images.githubusercontent.com/43263401/112616782-2f207800-8e35-11eb-8a81-9265ff4c92bf.png)

![image](https://user-images.githubusercontent.com/43263401/112616847-465f6580-8e35-11eb-8304-be7f5f7ad4aa.png)

![image](https://user-images.githubusercontent.com/43263401/112617277-cdacd900-8e35-11eb-9b2a-4e0e302e1d38.png)


I would like to suggest we add the images to the footer of the page then at the navbar we only retain the logout button

Reply:
If I understand the comment, it would look the same but internally be slightly different. I made a trial run of it and think it's ok to make this change BUT I worry about order of doing it. I would prefer to accept this PR and make the change as part of 0.5-rc changes. My logic is that I did make a v0.4.4 github release and it is 'ahead' of 0.5-rc. Accepting this PR gets 0.5-rc in sync with v0.4.4, and then we can make the change to .5 without having to make another 0.4.

Reply:
Thats okay by me